### PR TITLE
feat: Text utilities and list

### PR DESCRIPTION
**Pull request by Chakravarthi0**
added list component and text utilities
added utility classes for color

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
hey saran just a suggestion add heading and subheading to text utilities like you have done for list

File: components/text_utilities/text_utilities.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+.text-left {
+    text-align: start;
+}
+
+.text-right {
+    text-align: end;
+}
+
+.text-small{
+    font-size: 13px;
```
Comment (srishti-maurya) [inline components/text_utilities/text_utilities.css]: maybe you may use rem here for responsiveness too

**Review by srishti-maurya — APPROVED** [review]

**Comment by Chakravarthi0** [comment]
> hey saran just a suggestion add heading and subheading to text utilities like you have done for list

Sure preetha, thanks for reviewing

File: components/text_utilities/text_utilities.css (hunk 1 above)
Comment (Chakravarthi0) [reply components/text_utilities/text_utilities.css:16]: Hey Srishiti, thanks for reviewing, I will change that.

Comment (Chakravarthi0) [reply components/text_utilities/text_utilities.css]: Hey, thanks for reviewing, I will change that.

**Pull request merged**